Rules.md

```diff
@@ -80,7 +79,7 @@ Token types:
 * 2 x Train troops
     * +2 units in regions with a strategic city, otherwise +1 unit
 * 2 x Get money and invest ("invest" for short)
-    * Receive 2 coins and invest any amount (0-inf) in region
+    * Invest any amount (0-inf) in region
```

Comment:
Exactly. And that is more interesting, no? You always need to keep your army smaller than your investment if you want to keep growing. If you don't need growth anymore, then you can make one final push and have an army the same size as your investments.
You can however also go plundering to grow your investment.
Or you can rely on other players to fund your army by trading.
Or you can save money over multiple turns for when someone steals your investment.
Or you can train troops for one turn, and kill some troops some place else in the world at the end of the turn.
There are multiple strategies to deal with this.

I think it makes for a more interesting game.
In the version where you just receive 2 coins, you only need to put down the same number of investment tokens as train troops tokens and you will be fine.